AEPEdgeBridge.xcodeproj/project.pbxproj

```diff
@@ -22,10 +22,20 @@
 
 /* Begin PBXBuildFile section */
 		0334655BE881D0E2693A80C8 /* Pods_FunctionalTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 4B8946EAB205F842FB9989F1 /* Pods_FunctionalTests.framework */; };
+		2EB8E0332D77730F003ABDA2 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = D4D5B57C2432977F00CAB6E4 /* AppDelegate.swift */; };
+		2EB8E0342D77730F003ABDA2 /* SceneDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = D4D5B57E2432977F00CAB6E4 /* SceneDelegate.swift */; };
+		2EB8E0352D77730F003ABDA2 /* AssuranceView.swift in Sources */ = {isa = PBXBuildFile; fileRef = BF4A717826DDAE4B00612434 /* AssuranceView.swift */; };
+		2EB8E0362D77730F003ABDA2 /* ContentView.swift in Sources */ = {isa = PBXBuildFile; fileRef = D4D5B5802432977F00CAB6E4 /* ContentView.swift */; };
+		2EB8E0382D77730F003ABDA2 /* AEPEdgeBridge.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D444995F2519506B0093B364 /* AEPEdgeBridge.framework */; };
+		2EB8E03C2D77730F003ABDA2 /* Preview Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = D4D5B5852432977F00CAB6E4 /* Preview Assets.xcassets */; };
+		2EB8E03D2D77730F003ABDA2 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = D4D5B5822432977F00CAB6E4 /* Assets.xcassets */; };
+		2EF8F2952D78C08900EEBE04 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 2EF8F2932D78C08900EEBE04 /* LaunchScreen.storyboard */; };
 		49FD17161BDF671E27C6E44D /* Pods_UnitTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = CBE94B923FF16061DAFA46C6 /* Pods_UnitTests.framework */; };
 		4C089BCA2CF54A8A00CB500C /* FunctionalTestConstants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4C089BC92CF54A8A00CB500C /* FunctionalTestConstants.swift */; };
-		54AC083B374F80848671988A /* Pods_TestAppSwiftUI.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 37582C0001AF4867A5B12392 /* Pods_TestAppSwiftUI.framework */; };
+		54AC083B374F80848671988A /* (null) in Frameworks */ = {isa = PBXBuildFile; };
```

Comment:
Can you fix this in TestAppiOS/BuildDependencies so that null value does not show up?